WordPressVIPMinimum/Sniffs/Functions/RestrictedFunctionsSniff.php

```diff
@@ -223,6 +223,13 @@ public function getGroups() {
 					'lchown',
 				],
 			],
+			'filemtime' => [
+				'type'      => 'warning',
+				'message'   => 'VIP does not recommend %s() as each time a container is created, it would result in a different value for files in that container. See https://docs.wpvip.com/technical-references/vip-platform/file-concatenation-and-minification/ for more info.',
```

Comment:
```suggestion
				'message'   => 'VIP discourages the use of %s(), as each time a container is created, it would result in a different value for files in that container. See https://docs.wpvip.com/technical-references/vip-platform/file-concatenation-and-minification/ for more info.',
```